testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js

```diff
@@ -165,6 +165,58 @@ QUnit.module('Scenarios', moduleConfig, () => {
         });
     });
 
+    QUnit.test('Export [string x string/string(c1,c2) x number] & data_1.dataField_exist & data_2.dataField_not_exist with caption', function(assert) {
+        const done = assert.async();
+        const ds = {
+            fields: [
+                { area: 'row', dataField: 'row1' },
+                { area: 'column', dataField: 'col1' },
+                { area: 'data', dataType: 'number', dataField: 'data', summaryType: 'sum', caption: 'Sum' },
```

Comment:
By default these captions are 'Data (Sum)'/ 'Data (Avg)'

Reply:
Yes. I meant to do that.